README.md

```diff
@@ -1,3 +1,34 @@
-# Uploadcare Binding for PubNub Chat Engine
+# Uploadcare Plugin for Chat Engine
 
-Easily add image uploads to chatrooms using UploadCare online service.
+Adds the ability to upload images in ChatEngine.Chats via UploadCare service
+
+### Quick Start
+
```

Comment:
I think there also needs to be a link to Uploadcare setup webpage here so people know how to add the uploadcare part of the plugin. Or we bundle their upload jazz into this plugin. WDYT?